examples/device-a.py

```diff
@@ -0,0 +1,13 @@
+from rcluster import Device
+
+device = Device('device-a', ('224.0.0.64', 22464))
```

Comment:
Would it be possible at some point to have the ip address and port automatically configured? Would it matter if all the devices are on the same IP address and port? Just trying to find ways to make the interface is trivial to use as possible, and if every device has a unique name, we don't really care what the ip address is.

Reply:
All of the devices must be on the same ip/port as this is the multicast 'group'. All the other ip addresses and ports are automatically configured.

Reply:
The multicast group is essentially where all the pub/sub messages are being sent to and received from. It is also where requests will be sent to and devices listening for that request will have the option to respond directly to the requester.